.travis.yml

```diff
@@ -41,17 +38,15 @@ install:
 script:
   # Lint text-like files
   - scripts/pre-commit.sh --require-all
-  - make vendor-install
+  # Fetch vendor dependencies so linters succeed
+  - dep ensure -v -vendor-only
   # Lint golang sources
   - >
     set -o pipefail;
     gometalinter -j1 --sort=path --sort=line --sort=column
     --exclude="method NodeGetId should be NodeGetID"
     --deadline 9m --vendor --debug ./...
     |& stdbuf -oL awk '/linter took/ || !/^DEBUG/'
-  # Build the code & run tests
-  - make
-  - make test
```

Comment:
why other tests are removed from travis build ? @JohnStrunk

Reply:
The build is removed because that happens in the container now.

Unit tests via `100-gotest.sh` is run during the 1st stage. That leaves:
- testtest: doesn't do anything
- gofmt: I ~could~ should enable this in gml.
- golint: it's part of gml
- gometalinter: we've got that run manually (with way more options)
- test-makefiles: doesn't do much... we should get rid of the Makefile anyway
- test-shellscripts: calls shellcheck which is done by pre-commit.sh earlier
- govet: part of gml

I will add gofmt to gml, then I think we're covered.

Reply:
test-shellscripts also does a basic script validation using bash. It also does some things to make the output easier to read. I would at least keep that one.

Reply:
Added `bash -n` to the pre-commit.sh script.

Reply:
latest patch set has `gofmt` in gml. so, I am fine with this change.